vector/src/main/java/im/vector/app/features/home/room/detail/timeline/item/MergedRoomCreationItem.kt

```diff
@@ -227,6 +229,7 @@ abstract class MergedRoomCreationItem : BasedMergedItem<MergedRoomCreationItem.H
             val hasEncryptionEvent: Boolean,
             val isEncryptionAlgorithmSecure: Boolean,
             val roomSummary: RoomSummary?,
+            val canAddPeople: Boolean = false,
```

Comment:
Please rename to `canInvite` as per the other similar variable name for this feature.